### ISSUE TITLE
train VOC dataset

### ISSUE THREAD

**Issue by wudi00** [amdegroot/ssd.pytorch#386]
I train VOC dataset wit SSD 300, but loss is too high, why?
I have modified the batchsize and learning rate.
`iter 1680 || Loss: 1885.2950 || timer: 0.1291 sec.
iter 1690 || Loss: 2257.7227 || timer: 0.1306 sec.
iter 1700 || Loss: 1809.1299 || timer: 0.1481 sec.
iter 1710 || Loss: 1642.0751 || timer: 0.1394 sec.
iter 1720 || Loss: 1866.5345 || timer: 0.1319 sec.
iter 1730 || Loss: 2454.3920 || timer: 0.1527 sec.
iter 1740 || Loss: 2005.4082 || timer: 0.1345 sec.
iter 1750 || Loss: 1864.5095 || timer: 0.1418 sec.
iter 1760 || Loss: 2501.2754 || timer: 0.1366 sec.
iter 1770 || Loss: 1213.5183 || timer: 0.1323 sec.
iter 1780 || Loss: 1319.9740 || timer: 0.1283 sec.
iter 1790 || Loss: 1813.3102 || timer: 0.1406 sec.
iter 1800 || Loss: 1719.5035 || timer: 0.1365 sec.
iter 1810 || Loss: 2439.4969 || timer: 0.1342 sec.
iter 1820 || Loss: 1598.4881 || timer: 0.1377 sec.
iter 1830 || Loss: 1470.9045 || timer: 0.1405 sec.
iter 1840 || Loss: 1889.6567 || timer: 0.1430 sec.
iter 1850 || Loss: 1610.1531 || timer: 0.1399 sec.`

**Comment by yyx1107** [comment]
没遇到诶，我的是降的

**Comment by jaychou790118** [comment]
maybe trouble is your datasets.

**Comment by haozhihang123** [comment]
iter 0 || Loss: 25.6081 || timer: 0.0526 sec.
iter 10 || Loss: 39.6798 || timer: 0.0514 sec.
iter 20 || Loss: 148.1792 || timer: 0.0522 sec.
iter 30 || Loss: nan || timer: 0.0515 sec.
iter 40 || Loss: nan || timer: 0.0526 sec.
iter 50 || Loss: nan || timer: 0.0508 sec.
iter 60 || Loss: nan || timer: 0.0511 sec.
iter 70 || Loss: nan || timer: 0.0515 sec.
iter 80 || Loss: nan || timer: 0.0531 sec.
iter 90 || Loss: nan || timer: 0.0527 sec.
My result is like this. Has anyone ever been in this situation?

**Comment by Royee-Lyu** [comment]
> iter 0 || Loss: 25.6081 || timer: 0.0526 sec.
> iter 10 || Loss: 39.6798 || timer: 0.0514 sec.
> iter 20 || Loss: 148.1792 || timer: 0.0522 sec.
> iter 30 || Loss: nan || timer: 0.0515 sec.
> iter 40 || Loss: nan || timer: 0.0526 sec.
> iter 50 || Loss: nan || timer: 0.0508 sec.
> iter 60 || Loss: nan || timer: 0.0511 sec.
> iter 70 || Loss: nan || timer: 0.0515 sec.
> iter 80 || Loss: nan || timer: 0.0531 sec.
> iter 90 || Loss: nan || timer: 0.0527 sec.
> My result is like this. Has anyone ever been in this situation?

Decrease your learning rate
